data/transactions/logic/doc_test.go

```diff
@@ -77,6 +77,15 @@ func TestOpImmediateNote(t *testing.T) {
 	require.Empty(t, xd)
 }
 
+func TestAllImmediatesDocumented(t *testing.T) {
+	for _, op := range OpSpecs {
+		if len(op.Details.Immediates) > 0 {
+			xd := OpImmediateNote(op.Name)
+			require.NotEmpty(t, xd, "%s has undocumented immediates", op.Name)
```

Comment:
This is good enough for now, but ideally we'd have a test that confirms the two lengths are equivalent.

Reply:
I added a check that counts braces.  It has to be skipped for a few more complicated opcodes, but it's a nice addition for most.